docs/USAGE.md

```diff
@@ -1,52 +1,58 @@
-    + [skip_dir](#skip_dir)
-    + [skip_file](#skip_file)
-    + [skip_dotfiles](#skip_dotfiles)
-    + [skip_symlinks](#skip_symlinks)
-    + [monitor_interval](#monitor_interval)
-    + [min_notify_changes](#min_notify_changes)
-    + [Selective sync via 'sync_list' file](#selective-sync-via-sync_list-file)
-  * [Configuring the client for 'single tenant application' use](#configuring-the-client-for-single-tenant-application-use)
-  * [Configuring the client to use older 'skilion' application identifier](#configuring-the-client-to-use-older-skilion-application-identifier)
-  * [How to 'skip' directories from syncing?](#how-to-skip-directories-from-syncing)
-  * [How to 'rate limit' the application to control bandwidth consumed for upload & download operations](#how-to-rate-limit-the-application-to-control-bandwidth-consumed-for-upload--download-operations)
-  * [Shared folders (OneDrive Personal)](#shared-folders-onedrive-personal)
-  * [Shared folders (OneDrive Business or Office 365)](#shared-folders-onedrive-business-or-office-365)
-  * [SharePoint / Office 365 Shared Libraries](#sharepoint--office-365-shared-libraries)
-- [Running 'onedrive' in 'monitor' mode](#running-onedrive-in-monitor-mode)
-- [Running 'onedrive' as a system service](#running-onedrive-as-a-system-service)
-  * [OneDrive service running as root user via init.d](#onedrive-service-running-as-root-user-via-initd)
-  * [OneDrive service running as root user via systemd (Arch, Ubuntu, Debian, OpenSuSE, Fedora)](#onedrive-service-running-as-root-user-via-systemd-arch-ubuntu-debian-opensuse-fedora)
-  * [OneDrive service running as root user via systemd (Red Hat Enterprise Linux, CentOS Linux)](#onedrive-service-running-as-root-user-via-systemd-red-hat-enterprise-linux-centos-linux)
-  * [OneDrive service running as a non-root user via systemd (All Linux Distributions)](#onedrive-service-running-as-a-non-root-user-via-systemd-all-linux-distributions)
-  * [OneDrive service running as a non-root user via systemd (with notifications enabled) (Arch, Ubuntu, Debian, OpenSuSE, Fedora)](#onedrive-service-running-as-a-non-root-user-via-systemd-with-notifications-enabled-arch-ubuntu-debian-opensuse-fedora)
-  * [OneDrive service running as a non-root user via runit (antiX, Devuan, Artix, Void)](#onedrive-service-running-as-a-non-root-user-via-runit-antix-devuan-artix-void)
-- [Additional Configuration](#additional-configuration)
-  * [Using multiple OneDrive accounts](#using-multiple-onedrive-accounts)
-  * [Access OneDrive service through a proxy](#access-onedrive-service-through-a-proxy)
-  * [Setup selinux for a sync folder outside of the home folder](#setup-selinux-for-a-sync-folder-outside-of-the-home-folder)
-- [All available commands](#all-available-commands)
+- [Configuration and Usage of the OneDrive Free Client](#configuration-and-usage-of-the-onedrive-free-client)
+  - [Table of Contents](#table-of-contents)
```

Comment:
My editor decided to unify the Table of Contents section by replacing `*` with `-`. Let me know if I should change them back.